run.py

```diff
@@ -54,30 +54,12 @@ def run(
 
 
 if __name__ == "__main__":
-
     metrics = classification_metrics + calibration_metrics
-    hate_speech_data = transform_dataset(
-        load_dataset("tweet_eval", "hate"), preprocess_config
-    )
-    
-    run_configs = [
-        RunConfig(
```

Comment:
these are the ones that I moved to the library, so I can more freely change the dataset, the number of RunConfigs that I need, and the name. check out why!